Content.Server/Medical/SuitSensors/SuitSensorSystem.cs

```diff
@@ -205,7 +212,13 @@ private void OnVerb(EntityUid uid, SuitSensorComponent component, GetVerbsEvent<
             return;
 
         // standard interaction checks
-        if (!args.CanAccess || !args.CanInteract || args.Hands == null)
+        if (!args.CanInteract || args.Hands == null)
+            return;
+
+        if (!_interactionSystem.InRangeUnobstructed(args.User, args.Target))
```

Comment:
isn't this just !args.CanAccess?

Reply:
> isn't this just !args.CanAccess?

CanAccess won't let you change suitsensor on other player. It checks being in same container (as i remember when did it), so i used check range obsructed instead of it

Reply:
Alright

Reply:
This is untrue? It specifically checks inrangeunobstructed first. Verbs should never be doing manual raycasts.

Reply:
> This is untrue? It specifically checks inrangeunobstructed first. Verbs should never be doing manual raycasts. Raycasts are expensive and you should never be doing these manually.

Uh, it just don't work if i check it with CanAccess. Like, verb just don't appear when you try to change it on another player, so i can't figure out how to do it with using CanAccess.

Reply:
does it even need CanAccess? 🤔 the strip window seems to auto-close if you go out of accessibility anyways

Reply:
> does it even need CanAccess? 🤔 the strip window seems to auto-close if you go out of physical access range anyways

Yeah, it need it in case you want to change sensors on clothes lying the ground. Without check you can change sensors on any distance.

Reply:
Can you debug and find out why it's happening as doubling the raycasts is too expensive.

Reply:
> Can you debug and find out why it's happening as doubling the raycasts is too expensive.

As i can see from remarks from Shared/Verbs/VerbEvents.cs - CanAccess is combination of InRangeObstructed and IsInSameOrParentContainer. Because it checks same container, we can't change suit on another player. 

And i can't find where double raycasts is using. I replaced CasAccess with just InRangeObstructed check.

Reply:
> > Can you debug and find out why it's happening as doubling the raycasts is too expensive.
> 
> As i can see from remarks from Shared/Verbs/VerbEvents.cs - CanAccess is combination of InRangeObstructed and IsInSameOrParentContainer. Because it checks same container, we can't change suit on another player.
> 
> And i can't find where double raycasts is using. I replaced CasAccess with just InRangeObstructed check, so it raycasts once if i understand it right.